Rakefile

```diff
@@ -20,15 +18,9 @@ end
 
 task :default => [:clobber, :compile, 'README.md', :test]
 
-file 'README.md' => ['lib/debug/session.rb', 'lib/debug/config.rb',
```

Comment:
why do you remove this dependencies?

Reply:
Sorry that I didn't notice it's for checking timestamps 🤦‍♂️ I've dropped this change.